projects/addon-preview/components/preview/mixin.style.less

```diff
@@ -0,0 +1,5 @@
+.preview-item() {
+    border-radius: 1rem;
+    background: rgba(104, 104, 104, 0.96);
```

Comment:
do we have similar color from our palette ?

Reply:
No, unfortunately
We use the same colors in the dialog, but not from our palette